signinapp/model.py

```diff
@@ -442,6 +481,26 @@ def is_active(cls):
     def total_time(self) -> timedelta:
         return sum((s.elapsed for s in self.stamps), start=timedelta())
 
+    def raw_funds_for(self, user: User) -> float:
+        "Calculate funds from an event for the given user"
+        if not user.role.receives_funds:
+            return 0.0
+        user_stamps = user.stamps_for_event(self)
+        user_hours = sum((stamp.elapsed for stamp in user_stamps), start=timedelta())
+        total_hours = sum(
+            (stamp.elapsed for stamp in self.stamps if stamp.user.role.receives_funds),
+            start=timedelta(),
+        )
+        user_proportion = (user_hours / total_hours) if total_hours else 0.0
+        return user_proportion * (1 - self.overhead) * self.net_funds / 100.0
+
+    def funds_for(self, user: User) -> str:
+        return locale.currency(self.raw_funds_for(user))
+
+    @property
+    def overhead_funds(self) -> str:
+        return locale.currency(self.funds * self.overhead / 100.0)
```

Comment:
```suggestion
        return locale.currency(self.net_funds * self.overhead / 100.0)
```
This should account for the event cost